src/bin/pgcopydb/summary.c

```diff
@@ -750,64 +769,71 @@ void
 print_toplevel_summary(Summary *summary)
 {
 	char *d10s = "----------";
+	char *s10s = "          ";
```

Comment:
I assumed `d10s` stands for `dashes 10 string` and created `s10s` that should stand for `space 10 string`. If this naming does not make sense to other readers, feel free to suggest an alternative name.

I might as well use an empty string, as the `%10s` format would print required number of space characters. I think that seeing those spaces on the screen, near the beginning of the function helps readability.